### PR TITLE
Added installation instructions for client

### LINKED ISSUE #103: Add installation instructions for client
### What is the expected enhancement?

Add installation instruction for client folder

### PR DESCRIPTION
**Pull request by mentesniker**
### Summary
Added installation instructions  for client code

### Details and comments

Closes https://github.com/IceKhan13/purplecaffeine/issues/103

### CODE REVIEW

File: client/README.md [hunk 1]
```diff
@@ -69,6 +70,18 @@ Widget(local_storage).show()
...
 
+### Source Code Installation
+
+
+If you want to install from source code, execute while in the client folder of the project:
+
+
+```shell
+python3 setup.py install
+```
```
Comment (mickahell) [inline client/README.md]:
```suggestion
If you want to install from source code, execute while in the root folder of the project:

```shell
pip install client/.
```
```

**Review by mickahell — COMMENTED** [review]
Maybe better like this

**Review by IceKhan13 — CHANGES_REQUESTED** [review]
Can we do, something like

### Installation

Using pip:
```shell
pip install purplecaffeine
```

From sourcecode:
```shell
cd client
pip install .
```

? it will cover both cases

**Review by mentesniker — COMMENTED** [review]
I added additional instructions on the qucikstart section to include pip install purple caffeine

File: client/README.md [hunk 2]
```diff
@@ -69,6 +75,18 @@ Widget(local_storage).show()
 
 ----------------------------------------------------------------------------------------------------
 
+### Source Code Installation
+
+
+If you want to install from source code, execute while in the root folder of the project:
+
+```shell
+cd ./client
```
Comment (mickahell) [inline client/README.md]:
```suggestion
cd client
```
A cd with ./ is very uncommon

**Review by mickahell — CHANGES_REQUESTED** [review]

**Review by IceKhan13 — APPROVED** [review]
Thank you!

**Pull request merged**